### PR TITLE
Create jaiswalshash.json

### PR DESCRIPTION
**Pull request by jaiswalshash**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5863"></a>

### CODE REVIEW

File: data/jaiswalshash.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Shashank Jaiswal",
+  "type": "personal",
+  "bio": "Existing in a spider-verse where coding and sketching meet"
```
Comment (khushalarora) [inline data/jaiswalshash.json:4]: Please add a **' Comma '** at the end of this line :+1: 
```Suggestion
  "bio": "Existing in a spider-verse where coding and sketching meet",
```

File: data/jaiswalshash.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+{
+      "name": "Follow me on GitHub",
+      "url": "https://www.github.com/jaiswalshash",
+      "icon": "FaGithub"
+    },
+{
+      "name": "LeetCode",
+      "url": "https://www.leetcode.com/jaiswalshash/",
+      "icon": "FaLeetcode"
+    },
```
Comment (khushalarora) [inline data/jaiswalshash.json:25]: Please replace the **'Comma'** with a **' Square Bracket **]** '** in this line. Thank you :+1: 
```Suggestion
    }]
```

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hi @jaiswalshash, thank you for contributing :tada:

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**